docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[data-quality-dash]]
+= Data Quality dashboard
+
+The Data Quality dashboard shows you whether your data is effectively mapped to the https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[Elastic Common Schema] (ECS). Successful {ref}/mapping.html[mapping] enables you to search, visualize, and interact with your data throughout {es-sec} and {kib}.
+
+image::images/data-qual-dash.png[The Data Quality dashboard]
+
+You can use the Data Quality dashboard to:
+
+* Check one or multiple indices (by default, the indices used by {es-sec}) for unsuccessful mappings, to help you identify problems.
+* View detailed information about the fields in checked indices.
+* Track unsuccessful mappings by creating a case or markdown report based on Data Quality results.
+
+NOTE: The Data Quality dashboard does not show data from cold or frozen {ref}/data-tiers.html[data tiers]. It also does not display data from remote clusters using cross-cluster search — to view data from another cluster, log in to that cluster's {kib} instance.
+
+.Requirements
```

Comment:
This information is a little hard to read in the list format. What do you think about creating a new section and moving the content into a table?

Reply:
I am open to it but leaning the other way. Curious to hear what Joe and Janeen think when they review.

Reply:
I'm open to removing the "Required for ___" bits as you suggest, but I think I like including them for any power users or wannabe-power users who are concerned with the "why". Janeen what do you think?